internal/pkg/skuba/addons/cilium.go

```diff
@@ -141,24 +141,24 @@ spec:
     spec:
       serviceAccountName: cilium
       initContainers:
-      - name: install-cni-conf
+      - name: install-cni-bin
         image: {{.CiliumImage}}
         command:
           - /bin/sh
           - "-c"
-          - "cp -f /etc/cni/net.d/10-cilium-cni.conf /host/etc/cni/net.d/10-cilium-cni.conf"
+          - "cp -f /usr/lib/cni/* /host/opt/cni/bin/"
         volumeMounts:
-        - name: host-cni-conf
-          mountPath: /host/etc/cni/net.d
-      - name: install-cni-bin
+        - name: host-cni-bin
+          mountPath: /host/opt/cni/bin/
```

Comment:
For transition purposes, shouldn't we ensure the binaries are in /opt/cni/bin and /usr/lib/cni/ on the host?

Reply:
seems this is only changing the order of execution of the init containers :thinking:

Reply:
Yes, see the commit message.

But I think there might be other things to deal with.

Reply:
@evrardjp talks about the other thing which comes out from cri-o upgrade. Newest versions of cri-o expect CNI binaries only in `/opt/cni/bin`. We were previously installing the CNI binary in `/usr/lib/cni` instead.

The mountPath `/host/opt/cni/bin` is confusing, because `host-cni-bin` in real points to `/usr/lib/cni`, as you can see here. I'm really sorry for that!

https://github.com/SUSE/skuba/blob/master/internal/pkg/skuba/addons/cilium.go#L283-L286

Reply:
ic, thank for the explanation